frontend/src/pages/Landing/index.js

```diff
@@ -0,0 +1,331 @@
+import React, { useState, useEffect } from 'react';
+import { Tag, Table, Alert } from '@trussworks/react-uswds';
+import { Helmet } from 'react-helmet';
+import moment from 'moment';
+import { Link } from 'react-router-dom';
+
+import Container from '../../components/Container';
+import { getReports } from '../../fetchers/activityReports';
+import 'uswds/dist/css/uswds.css';
+import '@trussworks/react-uswds/lib/index.css';
+import './index.css';
+
+export const activityReportId = (id, regionId) => `R${
```

Comment:
I think this will eventually come from the server once https://github.com/HHS/Head-Start-TTADP/issues/145 is completed

Reply:
We could implement it on the server if needed.